app/controllers/catalog_controller.rb

```diff
@@ -379,7 +379,7 @@ def st_edit
   def st_form_field_changed
     return unless load_edit("st_edit__#{params[:id]}", "replace_cell__explorer")
     @group_idx = false
-    default_entry_point("generic") if params[:display]
+    default_entry_point("generic", "composite") if params[:display]
```

Comment:
@gmcculloug could you please review this change ? according to your comment on https://github.com/ManageIQ/manageiq/pull/11103#issuecomment-245697805 arguments could be ```(nil, "composite")``` - though with ```nil```, the line https://github.com/ManageIQ/manageiq/blob/master/app/controllers/catalog_controller.rb#L857 would fail (it doesn't expect ```nil``` there)
Also I'm not sure if the default entry point value(s) should be set always for new Catalog Bundle, or after the checkbox (Display in Catalog) has been checked ?